src/js/Beepers.js

```diff
@@ -1,4 +1,4 @@
-'use strict';
+import { bit_check, bit_clear, bit_set } from './serial_backend';
```

Comment:
This should go in utils - but out of scope of this PR.

Reply:
Definitely. Once modules are done I will go and do another pass. This highlighted quite a few circular dependencies which can we solved with simple re-org of files/modules.